README.md

```diff
@@ -2,56 +2,66 @@
 
 The repo contains the API server and client for retrieving hardware information.
 
-
 ## Prerequisites for Deploying Locally
 
-* Install [docker](https://docs.docker.com/engine/install/ubuntu/) and [setup permissions](https://docs.docker.com/engine/install/linux-postinstall/) for server deployment
-* Install `rust` and `cargo` for client deployment (https://www.rust-lang.org/tools/install)
-
+- Install [docker](https://docs.docker.com/engine/install/ubuntu/) and
+  [setup permissions](https://docs.docker.com/engine/install/linux-postinstall/)
+  for server deployment <<<<<<< HEAD
```

Comment:
Probably you forgot to delete it after rebasing

Reply:
Oops, fixed